nomos-services/cryptarchia-consensus/src/lib.rs

```diff
@@ -937,7 +1034,7 @@ where
         output
     }
 
-    fn validate_block(
+    fn validate_blocks_blobs(
```

Comment:
Block validation will eventually check more stuff than just the blobs, revert to `validate_blobs`

Reply:
Right, but I assume this is the function that only abstracts this specific validation. I created a higher level `validate_received_block` to group all validations (which is the one called from within `process_block`).
Should I still revert it?

Reply:
> Right, but I assume this is the function that only abstracts this specific validation. I created a higher level `validate_received_block` to group all validations (which is the one called from within `process_block`). Should I still revert it?

Ah, I missed that. That should be ok then.